test/Interop/Cxx/class/move-only/move-only-cxx-value-type.swift

```diff
@@ -76,4 +76,14 @@ MoveOnlyCxxValueType.test("Test move only field access in derived holder") {
 }
 #endif
 
+MoveOnlyCxxValueType.test("Test move only types behind smart pointers") {
```

Comment:
Hmm, it is in the intersection of the two. That being said, that test is disabled for windows. I just invoked swift-ci for this PR. If windows passes, I think keeping the test here makes sense, because we get more coverage. If the ci fails, I'll move it over to that file.

Reply:
Looks like the windows CI failed, so I'll move the tests over.